### ISSUE TITLE
Bug: Front Entrance Steps Slippery When Wet

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#212]
So far nobody has slipped on the steps, but they are _definitely_ slippery when wet. ⚠️ 
![image](https://user-images.githubusercontent.com/194400/206378267-3e035a5e-3f79-4f4e-91cb-384d18e96f19.png)

![image](https://user-images.githubusercontent.com/194400/206380459-f70ae0f1-1ccb-4b3a-a644-74ca1c35d2d7.png)

# Todo

+ [x] Discuss available options
+ [x] Select a solution
+ [x] Apply the fix

# FYI

Yesterday when I went to Leroy Merlin to collect the new Oil Heater,
I took the opportunity to speak to the manager of the paints & adhesives section.
Explained to her that our steps are polished granite [i.e. non-porous] and therefore _very_ slippery when wet. 
She informed me that the non-slip paint would not work because the paint would not bind/bond to the surface.
So she suggested the Tesa Anti-slip Tape:

![image](https://user-images.githubusercontent.com/194400/206380153-a0e43f55-f657-4f3d-8b31-2fc128276300.png)

It's basically the same as Grip Tape on a skateboard. 
To me, the only downside of this tape is the appearance. 🙄

**Comment by nelsonic** [comment]
No reply (objections) so I requested that our handyperson install the tape cause it was a sunny day:

![image](https://user-images.githubusercontent.com/194400/217497975-feecc2f3-954c-4341-b08d-c571fceeb24b.png)

The steps were pressure washed and allowed to fully dry before applying the tape. 
Expected useful life is 5 years. But I'll be happy with 2. Just stoked that we have _something_ in place.
So if there are older (and younger!) visitors to @home who are less sure on their feet ...
we can show that we've made an _effort_ to mitigate a potential slip. ✅
